service/src/main/java/greencity/service/AchievementServiceImpl.java

```diff
@@ -77,7 +78,7 @@ public AchievementVO save(AchievementPostDto achievementPostDto) {
             }
             userVO.getUserAchievements().add(userAchievementVO);
             userAchievementVO.setUser(userVO);
-            restClient.save(userVO);
+            userService.save(userVO);
```

Comment:
should the tests also be updated?

Reply:
added verify, everything else work correctly